lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -255,6 +260,44 @@ def compile_subconstants(name, constant)
           "\n" + output.join("\n\n")
         end
 
+        sig { params(value: T.untyped).returns(String) }
```

Comment:
`sig { params(value: Class).returns(String) }` ?

Reply:
This is outdated